docs/source/markdown/podman.1.md

```diff
@@ -1,56 +1,52 @@
 % podman(1)
 
 ## NAME
-podman - Simple management tool for pods, containers and images
+podman - simple management tool for pods, containers and images
 
 ## SYNOPSIS
-**podman** [*options*] *command*
+**podman command** [*optional*] *value*
```

Comment:
I think I have to clarify that the MANPAGE_SYNTAX is just to get an idea of the syntax. The synopsis was correct in this case.

Reply:
Although correct, I think it would be much nice to maintain a style! Lmk and I will change it back

Reply:
I think the previous version is better here. You would use commands like `podman --log-level debug ps` so `**podman** [*options*] *command*` is better IMO